src/VahterBanBot/Bot.fs

```diff
@@ -531,6 +531,13 @@ let justMessage
                     // not a spam
                     ()
             | None ->
+                let shouldDelete =
+                    message.Entities
+                     |> Array.exists (fun x -> (x.Type = Enums.MessageEntityType.TextMention || x.Type= Enums.MessageEntityType.Mention)  && x.Length = 0)
```

Comment:
Mow wait a minute. Do you know that the actual spammy mentions usually have length of 1?